src/core.c/CompUnit/Repository/FileSystem.pm6

```diff
@@ -12,13 +12,16 @@ class CompUnit::Repository::FileSystem
     has $!precomp-store;
     has $!distribution;
     has $!files-prefix;
-
-    my constant @extensions = <.rakumod .pm6 .pm>;
+    has @.extensions = <.rakumod .pm6 .pm>;
 
     method TWEAK(--> Nil) {
         $!loaded-lock := Lock.new;
         $!seen-lock := Lock.new;
         $!seen := nqp::hash;
+
+        # turn ".rakumod .pm6" into ".rakumod", ".pm6"
+        # for e.g. file#extensions<.rakumod .pm6>#lib
+        @!extensions = @!extensions.words;
```

Comment:
I thought this would Just Work without the `.words` hack here, but it would treat `<.rakumod .pm6>` as a single extension instead of two extensions.

Reply:
Is this because it's in the setting? `raku -e 'class A { has @.a = <.rakumod .pm6>; method e() { .say for @!a } }; A.new.e'` prints two lines for me.

Reply:
Its because the logic for parsing include specs is not the same as like the Raku grammar. So when `file#extensions<.rakumod .pm6>#lib` is parsed the `< ... >` doesn't act quite like the Raku quoting construct and instead just acts as a opening and closing marker that contains a single thing item regardless of spaces, commas, etc.